src/codeu/chat/client/commandline/Chat.java

```diff
@@ -425,7 +428,34 @@ public void invoke(List<String> args) {
         System.out.println("    Exit the program.");
       }
     });
-
+    panel.register("m-auth", new Panel.Command()
+    {
+      @Override
+      public void invoke(List<String> args) 
+      {
+        if (args.size() < 2) 
+        {
+      	  System.out.println("ERROR: Command doesn't follow the format. Please use 'help' for more information.");
+      	} 
+        else
+        {
+          String user = args.get(0);
+          String para = args.get(1);
+          if(findUser(user) == null)
+          {
+            System.out.println("ERROR: No such user.");
+          }
+          else if(!para.equals("o") && !para.equals("u") && !para.equals("b"))
```

Comment:
Also, because members are a kind of user, can we please change the "u" rank to an "m" rank? It just makes more sense.